### PR TITLE
Bear: Add `new_result` property

### PR DESCRIPTION
**Pull request by underyx**

### CODE REVIEW

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala/pull/2238/commits/3fd0d44a735d7b3a53b324f45243e927562d1660 https://github.com/coala-analyzer/coala/pull/2238/commits/b746b05c4a4f882db36f3a3ed794ef26ab59c6d2

You will need a rebase though later.

**Comment by underyx** [comment]
reack 45ab94a940616b8c60882503521df826401816d5 aa03825fb5a0550aa8de396208e24a9acc791d76

File: docs/Users/Tutorials/Writing_Bears.rst [hunk 1]
```diff
@@ -104,9 +102,9 @@ the creative name CommunicationBear:
                 ui=user_input,
                 type=type(user_input)))
 
-            return [Result.from_values(message="A hello world result.",
-                                       origin=self,
-                                       file=filename)]
+            yield self.create_result(message="A hello world result.",
```
Comment (mtomwing) [inline docs/Users/Tutorials/Writing_Bears.rst]: Should this be `self.new_result`?

Comment (sils1297) [inline docs/Users/Tutorials/Writing_Bears.rst]: indeed! This is a bug @underyx

**Comment by sils1297** [comment]
https://github.com/coala-analyzer/coala/pull/2238/commits/aa03825fb5a0550aa8de396208e24a9acc791d76 needs work

**Comment by underyx** [comment]
Fixing this the first thing tomorrow morning. Nice find, Michael, thanks!

**Comment by underyx** [comment]
reack https://github.com/coala-analyzer/coala/pull/2238/commits/ffd6508cec45d231951c1763f115ab0015d3c97c

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**